backend/src/types/types.ts

```diff
@@ -2,10 +2,19 @@ export interface User {
   dn: string
 }
 
+export const RoleKind = {
+  MODEL: 'model',
```

Comment:
What about data cards? Should this be `entry` so it can be used for both or do we need a third kind for data cards?